Sources/ShopifyCheckoutKit/CheckoutWebView.swift

```diff
@@ -99,6 +110,13 @@ class CheckoutWebView: WKWebView {
 	func load(checkout url: URL) {
 		load(URLRequest(url: url))
 	}
+
+	private func dispatchPresentedMessage(_ checkoutDidLoad: Bool, _ checkoutDidPresent: Bool) {
+		if !presentedEventDidDispatch && (checkoutDidLoad && checkoutDidPresent) {
```

Comment:
Reversing the logic here would make it easier to read I think

```suggestion
		if (checkoutDidLoad && checkoutDidPresent) && !presentedEventDidDispatch {
```